test/common/testlib.py

```diff
@@ -1171,12 +1169,12 @@ def assert_pixels(self, selector: str, key: str,
             for layout in self.layouts:
                 if layout["name"] not in skip_layouts:
                     self.set_layout(layout["name"])
+                    time.sleep(wait_delay)
```

Comment:
This means even layouts which have no pixel issues will have to wait 0.5 seconds for nothing. That feels a bit expensive for a RTL specific issue.

Reply:
Ah, I read that wrong, we did already sleep in `assert_pixels_in_current_layout` called below.

But I wonder if this will introduce other flakes, seeing that `assert_pixels_in_current_layout` is also called (now without sleep).
```
test/verify/check-system-services:            b.assert_pixels_in_current_layout("#services-page", "text-filter-test")
test/verify/check-system-services:            b.assert_pixels_in_current_layout("#nav-system", pixel_label)
```

Reply:
Why do we use this assert_pixels_in_current_layout and not the assert_pixels? Do you maybe know?

Reply:
Also this wait_delay is not set in any of these two examples, so there are not affected.

Reply:
`wait_delay: float = 0.5)` => wait_delay=0.5 so it's set by default.

Reply:
oops right, so now we still sleep but before we set the RTL.

Reply:
Seems the test time is still around the same time: https://cockpit-logs.us-east-1.linodeobjects.com/pull-18172-20230122-103906-69df97bb-fedora-37/log.html

And 52 minutes here. So maybe I was bit over concerned?